Content/StructureResolver.php

```diff
@@ -167,10 +178,15 @@ private function getStructureData(StructureInterface $structure): array
             $created = $document->getCreated();
         }
 
-        $type = 'page';
+        $type = 'unknown';
         if (method_exists($structure, 'getContent') && method_exists($structure->getContent(), 'getTemplateType')) {
             // used for content-bundle to determine current template type
             $type = $structure->getContent()->getTemplateType();
+        } elseif ($document instanceof StructureBehavior) {
+            $type = $this->documentInspector->getMetadata($document)->getAlias();
```

Comment:
can we add a comment: `determine type for structure that is implemented in the SuluPageBundle or the SuluArticleBundle`?